generator.yml

```diff
@@ -91,6 +91,16 @@ allconfigs:
   - &x86_64_nx_am {<< : *x86_64_mn_am}
   - &x86_64_nx_an {<< : *x86_64_mn_an}
   - &x86_64_nx_ay {<< : *x86_64_mn_ay}
+  #                                       https://git.kernel.org/linus/28187dc8ebd938d574edfc6d9e0f9c51c21ff3f4
+  - &arm32_510_am {config: [allmodconfig, CONFIG_CPU_BIG_ENDIAN=n], << : *arm-triple, << : *kernel_modules}
+  - &arm32_510_an {<< : *arm32_mn_an}
+  - &arm32_510_ay {config: [allmodconfig, CONFIG_CPU_BIG_ENDIAN=n], << : *arm-triple, << : *kernel_modules}
+  - &arm64_510_am {config: allmodconfig, << : *arm64-triple, << : *kernel_modules}
+  - &arm64_510_an {<< : *arm64_mn_an}
+  - &arm64_510_ay {config: allyesconfig, << : *arm64-triple, << : *kernel_modules}
+  - &x86_64_510_am {config: allmodconfig, << : *kernel_modules}
+  - &x86_64_510_an {<< : *x86_64_mn_an}
+  - &x86_64_510_ay {config: allyesconfig, << : *kernel_modules}
```

Comment:
I really think we should avoid distinct configs between different trees+branches.  This feels like a layering violation; suddenly our list of configs conflates a config with a certain branch of a certain tree.